code/src/layouts/best-sellers/BestSellers.test.tsx

```diff
@@ -17,6 +17,15 @@ jest.mock("@/store/api/productsApi", () => ({
   useGetProductsQuery: jest.fn()
 }));
 
+jest.mock("@/hooks/use-snackbar/useSnackbar", () => ({
+  __esModule: true,
+  default: jest.fn(() => ({ openSnackbar: () => {} }))
+}));
+
+jest.mock("@/store/api/cartApi", () => ({
+  useAddToCartMutation: jest.fn(() => [jest.fn(), {}])
+}));
```

Comment:
Same here (`jest.fn()`)

Reply:
The same thing, test fall without providing array as a mocked return value.